models/Request.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose")
+
+const requestSchema = new mongoose.Schema({
+    username:{
+        type:String,
+        require: true,
+        minlength:6,
+        maxlength:20,
+    },
+    pickupAddress:{
+        type:String,
+        require: true,
+        minlength:10,
+        maxlength:50,
+    },
+    dropAddress:{
+        type:String,
+        require: true,
+        minlength:10,
+        maxLength:50,
+    },
+    status:{
```

Comment:
instead of status let put a variable call isActive with type: boolean

Reply:
> instead of status let put a variable call isActive with type: boolean

I prefer to keep it the way it is. Status can be "pending" before it is accepted. Then it is "active" after it has been accepted by an assistant. And when it is completed status becomes "completed". So 3 states can't be captured by boolean. This way we have a record of completed requests for future use (auditing, statistics, etc...)